.eslintrc.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
   },
   plugins: ['react', '@typescript-eslint', 'prettier', 'jest', 'jsdoc'],
   parserOptions: {
-    ecmaVersion: 2020,
+    ecmaVersion: 2015,
```

Comment:
why downgrade?

Reply:
https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-large-snapshots.md#usage
> Because Jest snapshots are written with back-ticks (` `) which are only valid with ES2015 onwards you should set parserOptions in your config to at least allow ES2015 in order to use this rule:

Reply:
ooh wait that says atleast es 2015, let me re-check